py_lesson3_flask/main.py

```diff
@@ -0,0 +1,54 @@
+# Создать форму для регистрации пользователей на сайте. Форма должна содержать поля "Имя", "Фамилия",
+# "Email", "Пароль" и кнопку "Зарегистрироваться". При отправке формы данные должны сохраняться в базе
+# данных, а пароль должен быть зашифрован.
+
+
+from flask import Flask, render_template
+from data.users import User
+from data import db_session
+from flask_login import LoginManager
+from flask_wtf import FlaskForm
+from wtforms import PasswordField, StringField, SubmitField
+from wtforms.validators import DataRequired
+from werkzeug.utils import redirect
+
+app = Flask(__name__)
+app.config['SECRET_KEY'] = 'secret_key'
+login_manager = LoginManager()
+login_manager.init_app(app)
+
+
+class RegisterForm(FlaskForm):
+    name = StringField('Имя пользователя', validators=[DataRequired()])
+    last_name = StringField('Фамилия пользователя', validators=[DataRequired()])
+    email = StringField('Почта', validators=[DataRequired()])
```

Comment:
Можно использовать поле `EmailField`, но валидатор `Email()` тоже лучше добавить. Так как он проверяет еще и наличие домена.
```suggestion
    email = EmailField('Почта', validators=[DataRequired(), Email()])
```